### PR TITLE
docs(eol): add EOL options banner to header of every page

### PR DESCRIPTION
**Pull request by edclement**
# AngularJS Material is in LTS mode

We are no longer accepting changes that are not critical bug fixes into this project.
See [Long Term Support](https://material.angularjs.org/latest/#long-term-support) for more detail.

## PR Checklist

Please check your PR fulfills the following requirements:
- [ ] Does this PR fix a regression since 1.2.0, a security flaw, or a problem caused by a new browser version?
- [x] The commit message follows [our guidelines](https://github.com/angular/material/blob/master/.github/CONTRIBUTING.md#-commit-message-format)
- [x] Tests for the changes have been added or this is not a bug fix / enhancement
- [x] Docs have been added, updated, or were not required

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Enhancement
[x] Documentation content changes
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?
image

## What is the new behavior?
image

image

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

### CODE REVIEW

File: docs/config/template/index.template.html [hunk 1]
```diff
@@ -149,6 +149,10 @@ <h2 class="md-toolbar-item md-breadcrumb md-headline" tabindex="-1">
         </div>
       </div>
 
+      <div class="eol-notice">
+          AngularJS Material is now in Long Term Support (LTS) mode. Explore End Of Life (EOL) options <a href="https://blog.angular.io/finding-a-path-forward-with-angularjs-7e186fdd4429">here</a>.
```
Comment (Splaktar) [inline docs/config/template/index.template.html:153]: The `Long Term Support (LTS) mode` should probably link to `/#long-term-support`.

**Review by Splaktar — APPROVED** [review]
LGTM other than adding a link to our LTS notice. It would also be helpful to post a screenshot of this banner on a demo or API page.

File: docs/config/template/index.template.html (hunk 1 above)
Comment (edclement) [reply docs/config/template/index.template.html]: done

**Comment by edclement** [comment]
> LGTM other than adding a link to our LTS notice. It would also be helpful to post a screenshot of this banner on a demo or API page.

image

image

**Review by crisbeto — APPROVED** [review]

**Pull request merged**